docs/topics/developers.md

```diff
@@ -164,7 +164,7 @@ Here is an example script for running a quick test against a locally running bri
 
 export BRIGADE_EVENT_TYPE=quicktest
 export BRIGADE_EVENT_PROVIDER=script
-export BRIGADE_COMMIT=9c75584920f1297008118915024927cc099d5dcc
+export BRIGADE_COMMIT_ID=9c75584920f1297008118915024927cc099d5dcc
```

Comment:
Shouldn't we include `BRIGADE_COMMIT_REF` rather than `~ID` because it is `REF` which value always exists?

Reply:
And you already seem to be aware of that in [selftest.sh](https://github.com/Azure/brigade/pull/307/files#diff-14d431b5ed62d9f4ec854c1e155124a9R16)!